### PR TITLE
Fix the issue of validating Server_address of Syslog destination, upd…

### PR DESCRIPTION
**Pull request by Abhishek-121**
…ate description, port validation in syslog, webhook headers update.

In this commit --

-- Fix the issue of validating server address while configuring syslog destination.

-- Address the issue of validating port number between ranges and different datatypes.

-- Update the description of syslog destination in the documentation.

-- Update the headers as per latest sdk changes while configuring the webhook destination.

### CODE REVIEW

File: plugins/modules/events_and_notifications_workflow_manager.py [hunk 1]
```diff
@@ -301,6 +301,8 @@
 - python >= 3.5
 
 notes:
+  - Configuring webhook destination with headers start supporting from dnacentersdk 2.9.1 onwards as we are aligned with CatC Release 2.3.7.5.
```
Comment (madhansansel) [inline plugins/modules/events_and_notifications_workflow_manager.py:304]: Catalyst Center

File: plugins/modules/events_and_notifications_workflow_manager.py [hunk 2]
```diff
@@ -301,6 +301,8 @@
...
+  - Configuring Syslog destination start supporting from dnacentersdk 2.9.x onwards as we are aligned with CatC Release 2.3.7.5.
```
Comment (madhansansel) [inline plugins/modules/events_and_notifications_workflow_manager.py:305]: Catalyst Center

File: plugins/modules/events_and_notifications_workflow_manager.py (hunk 1 above)
Comment (madhansansel) [reply plugins/modules/events_and_notifications_workflow_manager.py:304]: Configuring the webhook destination with headers now supports starting from dnacentersdk version 2.9.1 onwards. This enhancement is in alignment with Catalyst Center Release 2.3.7.5.

Can we write the same for the below as well?

File: plugins/modules/events_and_notifications_workflow_manager.py [hunk 3]
```diff
@@ -2304,7 +2279,14 @@ def get_diff_merged(self, config):
                 self.log(self.msg, "ERROR")
                 return self
 
-            if not port.isdigit() or (isinstance(port, int) and port not in range(1, 65536)):
+            if isinstance(port, str):
+                if not port.isdigit() or (int(port) not in range(1, 65536)):
+                    self.status = "failed"
+                    self.msg = "Invalid Syslog destination port '{0}' given in playbook. Select port from the number range(1, 65535)".format(port)
```
Comment (madhansansel) [inline plugins/modules/events_and_notifications_workflow_manager.py:2285]: Please choose a port within the range of numbers (1, 65535)

File: plugins/modules/events_and_notifications_workflow_manager.py (hunk 1 above)
Comment (Abhishek-121) [reply plugins/modules/events_and_notifications_workflow_manager.py]: updated

**Review by madhansansel — APPROVED** [review]

File: plugins/modules/events_and_notifications_workflow_manager.py (hunk 2 above)
Comment (Abhishek-121) [reply plugins/modules/events_and_notifications_workflow_manager.py]: updated.

File: plugins/modules/events_and_notifications_workflow_manager.py (hunk 3 above)
Comment (Abhishek-121) [reply plugins/modules/events_and_notifications_workflow_manager.py]: updated.

**Pull request merged**